provider/cldr/src/cldr_paths.rs

```diff
@@ -15,7 +15,7 @@ pub trait CldrPaths: std::fmt::Debug {
 
     /// Path to checkout of cldr-dates:
     /// <https://github.com/unicode-cldr/cldr-dates-full>
-    fn cldr_dates(&self) -> Result<PathBuf, Error>;
+    fn cldr_dates_gregorian(&self) -> Result<PathBuf, Error>;
```

Comment:
Note: This function name is fine as "gregorian" because it is from CLDR JSON.  When passing around the string identifier for the calendar, though, we should use "gregory"